rollup.config.js

```diff
@@ -46,7 +52,7 @@ function createCommonJSConfig(input, output) {
     external,
     plugins: [
       resolve({ extensions }),
-      typescript(),
+      getEsbuild('es2015'),
```

Comment:
```suggestion
```

according to https://github.com/pmndrs/zustand/pull/343#issuecomment-808745679 @aulneau

Reply:
fixed!